### DISCUSSION TITLE
How to enable scrollabl (vertically) code block when the block exceeds maxHeight?

### DISCUSSION THREAD

**Discussion by jeremy-feng** [squidfunk/mkdocs-material#5133] [Q&A] [answered]
As shown on [https://fmc.feffery.tech/custom-code-block-style](https://fmc.feffery.tech/custom-code-block-style), the code block can have a limited `maxHeight`. If the code block contains too many lines of code, we will have a vertically scrollable bar.

I think this is a very helpful feature. Does anyone know how to do so? Thanks!

**Comment by squidfunk** [comment, marked as answer]
This has been asked several times. Please use the search... Duplicate of:

- #3444
- #4964
- #5082

**Reply by jeremy-feng** [reply]
Thanks so much! Sorry for my poor search ability (I didn't search for the appropriate key words 😭 ).
